AWS/IAM/TPlaybook_delete_IAMUser.md

```diff
@@ -0,0 +1,31 @@
+
+<img src="../../images/icons/Tamnoon.png" width="200"/>
+
+# Tamnoon Playbook: IAM - Delete IAMUser .
+
+## Description
+This playbook describes how to execute Tamnoon IAMHelper automation to delete iam user.
+
+The execution is based on AWS credentials configuration based on the next fallbacks:
+1. If AWS profile or aws access key and secret were given, use it as an AWS credentials source.
```

Comment:
Instead of "were", "are"

Reply:
I think i should just do a once-around language review...don't worry about it now